src/Shared/Recruit.Vacancies.Client/Infrastructure/TableStore/StorageTableNames.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Esfa.Recruit.Vacancies.Client.Infrastructure.TableStore
+{
+    internal static class StorageTableNames
+    {
+        internal const string TableName = "QueryStore";
```

Comment:
Name should indicate underline table name ... perhaps ... `StorageTableNames.QueryStoreName`